code/modules/mob/living/living_defines.dm

```diff
@@ -171,3 +172,12 @@
 	var/body_position_pixel_x_offset = 0
 	///The x amount a mob's sprite should be offset due to the current position they're in
 	var/body_position_pixel_y_offset = 0
+
+	/// FOV view that is applied from either nativeness or traits
+	var/fov_view
+	/// Native FOV that will be applied if a config is enabled
+	var/native_fov = FOV_90_DEGREES
```

Comment:
if you have a datum, why are you applying these settings on the mob level

Reply:
The datum is the handler for the client, and is only instantiated when a client enters that mob. 
You will want to manage those variables whether that handler exists or not.